ntp-proto/src/packet/extensionfields.rs

```diff
@@ -806,6 +873,32 @@ mod tests {
         );
     }
 
+    #[cfg(feature = "ntpv5")]
+    #[test]
+    fn draft_identification() {
+        let test_id = "draft-ietf-ntp-ntpv5-00\0";
+        let len = u16::try_from(4 + test_id.len()).unwrap();
+        let mut data = vec![];
+        data.extend_from_slice(&[0xF5, 0xFF]);
+        data.extend_from_slice(&len.to_be_bytes());
+        data.extend_from_slice(test_id.as_bytes());
```

Comment:
total nit: using `.extend` would work just as well here. A bit shorter and more common

Reply:
Done!